iOS_Labs/Utils/Photos/PhotoController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  PhotoController.swift
+//  iOS_Labs
+//
+//  Created by Daniil Oleshchuk on 29.11.2021.
+//
+
+import Foundation
+import Alamofire
+
+
+struct PhotoController {
+    private static var API_BASE_URL: String {
+        get {
+            guard let filePath = Bundle.main.path(forResource: "Unsplash-Info", ofType: "plist") else {
+                fatalError("Couldn't find file 'Unsplash-Info.plist'.")
+            }
+            let plist = NSDictionary(contentsOfFile: filePath)
+            guard let value = plist?.object(forKey: "API_BASE_URL") as? String else {
+                fatalError("Couldn't find key 'API_BASE_URL' in 'Unsplash-Info.plist'.")
+            }
+            return value
+        }
+    }
+    
+    private static var API_ACCESS_KEY: String {
+        get {
+            guard let filePath = Bundle.main.path(forResource: "Unsplash-Info", ofType: "plist") else {
+                fatalError("Couldn't find file 'Unsplash-Info.plist'.")
+            }
+            let plist = NSDictionary(contentsOfFile: filePath)
+            guard let value = plist?.object(forKey: "API_ACCESS_KEY") as? String else {
+                fatalError("Couldn't find key 'API_ACCESS_KEY' in 'Unsplash-Info.plist'.")
+            }
+            return value
+        }
+    }
+    
+    private static var API_SECRET_KEY: String {
```

Comment:
This kind of things is better to store in keychain. https://developer.apple.com/documentation/security/keychain_services/keychain_items/using_the_keychain_to_manage_user_secrets